### ISSUE TITLE
Random DNS problems during Docker build

### ISSUE THREAD

**Issue by lidel** [ipfs-shipyard/ipfs-companion#371]
While looking into #370, I noticed that `alpine` images are still non-deterministic when it comes to dns lookups (commands executed on the same box, one after another):

```
> docker run -it --rm node:9.5.0-alpine ping -c 1 nodejs.org
PING nodejs.org (104.20.23.46): 56 data bytes
64 bytes from 104.20.23.46: seq=0 ttl=59 time=21.822 ms

--- nodejs.org ping statistics ---
1 packets transmitted, 1 packets received, 0% packet loss
round-trip min/avg/max = 21.822/21.822/21.822 ms
```
```
$ docker run -it --rm node:9.5.0-alpine ping -c 1 nodejs.org
ping: bad address 'nodejs.org'
```
```
$ docker run -it --rm node:9.5.0-alpine ping -c 1 nodejs.org
PING nodejs.org (104.20.22.46): 56 data bytes
64 bytes from 104.20.22.46: seq=0 ttl=59 time=22.924 ms

--- nodejs.org ping statistics ---
1 packets transmitted, 1 packets received, 0% packet loss
round-trip min/avg/max = 22.924/22.924/22.924 ms
```
```
$ docker run -it --rm node:9.5.0-alpine ping -c 1 nodejs.org
ping: bad address 'nodejs.org'
```

what the.. 

If i force ipv4 lookups (`ping -4`) then it always succeeds.  
Seems that the problem surfaces only on boxes that default to ipv6 (like my fiber). 

ps. this _might_ be underlying source of related issues:
- https://github.com/rancher/rancher/issues/5041#issuecomment-297722504
- https://github.com/gliderlabs/docker-alpine/issues/8#issuecomment-360187102
- https://github.com/yarnpkg/yarn/issues/746

**TL;DR**

Why is this a problem? We want reproducible builds to work not only on Jenkins, but also on local machines (eg. on laptop of AMO reviewer). 

A fix is to find workaround for alpine, or switch to glibc-based image.

**Comment by lgierth** [comment]
A glibc-based image will allow you to run binaries from dist.ipfs.io (go-ipfs, etc.)

**Comment by lidel** [comment]
Closed by #372
